tests/e2e/set-ffi-env-e2e

```diff
@@ -37,6 +37,19 @@ source "${SCRIPT_DIR}"/lib/diskutils
 source "${SCRIPT_DIR}"/lib/repoutils
 
 # GLOBALS
+# Configuration Constants
+readonly BLUECHI_CONTROLLER_CONF="/etc/bluechi/controller.conf"
+readonly BLUECHI_AGENT_CONF="/etc/bluechi/agent.conf.d/00-default.conf"
+readonly QM_BLUECHI_AGENT_CONF="/etc/qm/bluechi/agent.conf.d/agent.conf"
+readonly QM_CONTAINER_DROPIN_DIR="/etc/containers/systemd/qm.container.d"
+readonly BLUECHI_SOCKET_DROPIN="${QM_CONTAINER_DROPIN_DIR}/bluechi-socket.conf"
+readonly BLUECHI_SOCKET_HOST="/run/bluechi/bluechi.sock"
+readonly BLUECHI_SOCKET_CONTAINER="/run/bluechi.sock"
+readonly DEFAULT_NODE_NAME="localrootfs"
+readonly DEFAULT_HOSTNAME="localrootfs"
```

Comment:
What is the difference between `DEFAULT_HOSTNAME` and `CONTROLLER_CONTAINER_NAME`? Both seems to be configuring the name of the agent on the controller partition. When is which used and why?

Reply:
Same for `DEFAULT_NODE_NAME` and `NODES_FOR_TESTING`